.github/workflows/_build_artifacts.yml

```diff
@@ -240,15 +227,15 @@ jobs:
           export_environment_variables: true
           create_credentials_file: true
       - name: Copy binaries to Google Cloud Storage
-        if: ${{ inputs.profile == 'release' && matrix.stage == 'release' && (matrix.name.image_name == 'gateway' || matrix.name.image_name == 'client') }}
+        if: ${{ inputs.profile == 'release' && matrix.stage == 'release' && contains(fromJSON('["relay", "gateway", "client"]'), matrix.name.image_name) }}
         run: |
           gcloud storage cp \
             ${BINARY_DEST_PATH} \
-            gs://firezone-staging-artifacts/${{ matrix.name.artifact }}/${{ env.VERSION }}-${{ inputs.sha }}/${{ matrix.arch.shortname }}
+            gs://firezone-staging-artifacts/firezone-${{ matrix.name.image_name }}/${{ inputs.sha }}/${{ matrix.arch.shortname }}
```

Comment:
Why do we need a prefix here? It's already in `firezone` bucket

Reply:
This maintains what we had before. `firezone-gateway` is used as the artifact name:

https://github.com/firezone/firezone/blob/main/terraform/modules/google-cloud/apps/gateway-region-instance-group/variables.tf#L97

I'm using the `image` because it simplifies the matrix in the deploy.yml workflow